omx/gstomxjpegenc.c

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Why is this element necessary? What's the difference with the mjpeg?

Reply:
This is the encoder element for MJPEG, it's a new element, the other mjpeg element is a decoder.